client/state/data-layer/wpcom/auth/send-login-email/index.js

```diff
@@ -66,6 +74,7 @@ export const sendLoginEmail = ( action ) => {
 					email: email,
 					...( redirect_to && { redirect_to } ),
 					...( flow && { flow } ),
+					create_account: createAccount,
```

Comment:
If we send this as true, the magic link class in WPCOM will create a new account for the given email.